examples/hello-world/main.c

```diff
@@ -20,13 +20,29 @@
  */
 
 #include <stdio.h>
+#include "xtimer.h"
+#include "pm_layered.h"
+
+/* use chars to test the alignment */
+static char counter __attribute__((section(".backup.data"))) = 8;
+static char counter_zero __attribute__((section(".backup.bss")));
 
 int main(void)
 {
     puts("Hello World!");
 
-    printf("You are running RIOT on a(n) %s board.\n", RIOT_BOARD);
-    printf("This board features a(n) %s MCU.\n", RIOT_MCU);
+    printf("counter value, %d\n", counter);
+    printf("counter_zero value, %d\n", counter_zero);
+
+    while (1) {
+        printf("You are running RIOT on a(n) %s board.\n", RIOT_BOARD);
+        printf("This board features a(n) %s MCU.\n", RIOT_MCU);
+        printf("counter value, %d\n", counter++);
+        printf("counter_zero value, %d\n", counter_zero++);
```

Comment:
Isn't better to create a dedicated test instead of temporarily hacked this one ?
This may be useful to newcomers that need this feature.

Reply:
Yes, of course (in fact it was mentioned in a short discussion on ben's branch). That, along with a documentation page.

Reply:
I've added a dedicated (albeit pretty simple) test.
How should I go about the documentation?